apps/studio/components/interfaces/Database/Functions/FunctionsList/FunctionList.tsx

```diff
@@ -109,15 +109,19 @@ const FunctionList = ({
             <TableCell className="truncate">
               <Button
                 type="text"
-                className="text-foreground text-sm p-0 hover:bg-transparent"
+                className="text-link-table-cell text-sm p-0 hover:bg-transparent title"
                 onClick={() => editFunction(x)}
+                title={x.name}
```

Comment:
`title` being [the HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Global_attributes/title) that adds a browser-native tooltip. Useful for the name instances of a TableCell, in case they are truncated.